src/System.Management.Automation/engine/hostifaces/MshHost.cs

```diff
@@ -64,9 +64,9 @@ protected PSHost()
         /// The name identifier of the hosting application.
         /// </value>
         /// <example>
-        ///     <MSH>
+        ///     <PowerShell>
         ///         if ($Host.Name -ieq "ConsoleHost") { write-host "I'm running in the Console Host" }
-        ///     </MSH>
+        ///     </PowerShell>
```

Comment:
`<PowerShell></PowerShell>` is not a valid XML element for ///-comment documentation.